### PR TITLE
[fix] #94: use execve instead of execvpe.

### PR DESCRIPTION
**Pull request by M5oul**

### CODE REVIEW

File: srcs/expansion/command_substitution.c [hunk 1]
```diff
@@ -67,6 +68,8 @@ static void					add_substitution(t_strlist **strlist_addr
 	extern char			**environ;
 	char				**environ_backup;
```
Comment (lsimonne) [inline srcs/expansion/command_substitution.c]: same here as in #134 , no need for extern environ + backup, just one env set to get_variable_for_execution

Comment (M5oul) [inline srcs/expansion/command_substitution.c]: Already done on https://github.com/42shpimanmls/42sh/commit/c1e40fc242ffd18afe65982562ffe40d56b06268

Comment (lsimonne) [inline srcs/expansion/command_substitution.c]: ah oui, sry!

**Comment by pilespin** [comment]
close

**Comment by lsimonne** [comment]
you forgot to reassign environ to get_variables_for_execution in add_subst (and to leave it!)

**Comment by M5oul** [comment]
Rah, `make` is not enough to test things.
Always need to do `make re` which is useless.

**Pull request merged**